terraform-scripts/main.tf

```diff
@@ -52,6 +52,6 @@ locals {
     # https://github.com/release-engineering/Sync2Jira/
     # https://github.com/fedora-infra/webhook-to-fedora-messaging/
     # https://issues.redhat.com/projects/GHQKIVERSE
-    sync2jira_redhat = "1137195"
+    sync2jira_redhat = "60685343"
```

Comment:
@gsmet told me the app ID was `1137195`?

Reply:
That's the installation ID (https://github.com/organizations/quarkiverse/settings/installations/60685343)

Reply:
Ok. Might want to update the comment above :)

```
  # Application IDs installed in the Quarkiverse organization
  # These applications are enabled on a per-repository basis
```

Reply:
Thanks for fixing it!